cellprofiler/modules/untangleworms.py

```diff
@@ -112,9 +112,19 @@
 from cellprofiler.preferences import standardize_default_folder_names, \
     DEFAULT_INPUT_FOLDER_NAME, DEFAULT_OUTPUT_FOLDER_NAME, NO_FOLDER_NAME, \
     ABSOLUTE_FOLDER_NAME
-from cellprofiler.modules._help import NAMING_OUTLINES_HELP, RETAINING_OUTLINES_HELP, USING_METADATA_GROUPING_HELP_REF, \
+from cellprofiler.modules._help import USING_METADATA_GROUPING_HELP_REF, \
```

Comment:
Does the NAMING_OUTLINES_HELP need to be replaced also?

Reply:
Nope. It was unused.